classes/table/report.php

```diff
@@ -75,6 +75,34 @@ private function format_uri($uri, $label = '', $size = 80) {
         return \html_writer::link($uri, $label);
     }
 
+    /**
+     * Format url allowed column.
+     *
+     * @param $record
+     * @return string
+     */
+    protected function col_urlallowed($record) {
+        if ((int)$record->urlallowed === 1) {
+            return "Yes";
```

Comment:
can you please use the moodle core lang strings for yes and no please, here and below